.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+go:
+- 1.11.x
```

Comment:
I suggest moving to, or just adding ```- 1.12.x```

Reply:
We were blocked on a Go 1.12 upgrade because of something breaking in go-ethereum until recently - quite possibly we're unblocked now, which would be great! I'll run a build and check.

Reply:
I believe that's fixed.  I built go-ethereum master branch on Saturday using go1.12.5

Reply:
Switch to go 1.12 on all platforms, seems to be working great!

Reply:
I can't name every single optimization, but it is significantly better everywhere i have it deployed.   I'm glad to hear the incompatibility you mentioned is gone :-D